### PR TITLE
WIP - Feature/layout components

### PR DESCRIPTION
**Pull request by farinalvaro**
This Pull Request solves the task #20, we created and styled new components in `rifui` library using [Materail UI](https://material-ui.com/) to be consumed in the src project.

## Known issues
- Some old components still depend on `react-bootstrap`, both the dependency and old components will be removed.
- Install `Rubik` font.
- Need to add responsive support for some pages.

### CODE REVIEW

File: src/rifui/theme/theme.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+export const colors = {
+	black: '#000000',
+	primary: '#008FF7',
+	darkBlue: '#197DC6',
+	gray1: '#F8F7F7',
+	gray2: '#E5E5E5',
+	gray3: '#C4C4C4',
+	gray4: '#919191',
+	gray5: '#4D4C4C',
+	transparent: '#FFFFFF00',
```
Comment (vojtechsimetka) [inline src/rifui/theme/theme.ts:12]: Wow I did not know browsers support this! Cool

File: src/rifui/components/atoms/forms/Grid.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+const Grid: FC<GridProps> = ({ children, ...rest }) => {
+  return (
+    <MUIGrid {...rest}>
+      {children}
+    </MUIGrid>
+  );
+}
+
+export default Grid;
```
Comment (jurajpiar) [inline src/rifui/components/atoms/forms/Grid.tsx:14]: Duplicated component (src/rifui/components/atoms/Grid.tsx)

File: src/rifui/components/atoms/forms/Input.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+const Input: FC<InputProps> = ({ children, ...rest }) => {
+	return (
+		<MUIInput {...rest}>
+			{children}
+		</MUIInput>
+	);
+};
+
+export default Input;
```
Comment (jurajpiar) [inline src/rifui/components/atoms/forms/Input.tsx:14]: Tab indented.

File: src/rifui/components/atoms/forms/InputAdornment.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+const InputAdornment: FC<InputAdornmentProps> = ({ children, ...rest }) => {
+	return (
+		<MUIInputAdornment {...rest}>
+			{children}
+		</MUIInputAdornment>
+	);
+};
+
+export default InputAdornment;
```
Comment (jurajpiar) [inline src/rifui/components/atoms/forms/InputAdornment.tsx:14]: Tab indented.

File: src/rifui/components/atoms/forms/InputLabel.tsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+const InputLabel: FC<InputLabelProps> = ({ children, ...rest }) => {
+	return (
+		<MUIInputLabel {...rest}>
+			{children}
+		</MUIInputLabel>
+	);
+};
+
+export default InputLabel;
```
Comment (jurajpiar) [inline src/rifui/components/atoms/forms/InputLabel.tsx:14]: Tab indented.

File: src/rifui/components/atoms/forms/TextField.tsx [hunk 6]
```diff
@@ -1,10 +1,21 @@
...
+import { colors } from 'rifui/theme';
 
 export type TextFieldProps = MUITextFieldProps;
 
-const TextField: FC<TextFieldProps> = props => {
-  return <MUITextField {...props} />;
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    color: colors.gray3,
+    '&:hover': {
```
Comment (jurajpiar) [inline src/rifui/components/atoms/forms/TextField.tsx:10]: nice!

File: src/rifui/components/atoms/forms/Grid.tsx (hunk 2 above)
Comment (jurajpiar) [reply src/rifui/components/atoms/forms/Grid.tsx:14]: removed this forms one as Grid belong more to layout than forms.

**Pull request merged**